### DISCUSSION TITLE
FastAPI Websocket

### DISCUSSION THREAD

**Discussion by tops-sdbs** [deta/space-docs#161] [Q&A] [answered]
How do I get the URL/endpoint, for WebSocket connection once I have deployed my python-server-side code (FastAPI) to deta space. So far I have only worked with normal POST, GET etc requests using HTTP protocol. So i am not sure about WebSocket connection and where to get the public url that i can connect with. 

my server-side-code (deployed to deta space):
![image](https://github.com/deta/space-docs/assets/135554395/550fb058-86ea-4ff0-aa52-e18131e242cf)

my client-side code:
![image](https://github.com/deta/space-docs/assets/135554395/dcdef9fe-5ae4-46eb-8dc5-78bb052f8a0d)

**Comment by fffena** [comment, marked as answer]
WebSockets aren't supported as micro can only handle up to 20 seconds.
https://deta.space/docs/en/build/faq#do-micros-support-websockets
